src/api.rs

```diff
@@ -491,7 +491,7 @@ impl Context {
     if self.fi.show_existing_frame {
       self.idx += 1;
 
-      let mut fs = FrameState::new(&self.fi, self.seq.chroma_sampling);
+      let mut fs = FrameState::new(&self.seq, &self.fi, self.seq.chroma_sampling);
```

Comment:
Maybe already answered in irc, but can you pls explain why fi info is required in fs?

Reply:
> why fi info is required
You mean `seq`?

Reply:
Oh, right, the seq.